docs/competition/matches/diagrams/arena.svg

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Don't bother reviewing this file, it's copied directly from the rulebok

Reply:
Confirmed.